components/ILIAS/Contact/PRIVACY.md

```diff
@@ -0,0 +1,87 @@
+  member/tutor relations or test correction processes, etc
+- The global activation of the component can be checked with the
+  "Administration > Communication > Contact > Activate 'Contacts'" setting.
+- The personal activation of the component can be toggled with the
+  "Profile and Privacy > Visibility > Allow Contact Requests" setting.
+- Contacts can be requested within member lists, like in courses and groups.
+- Furthermore, when a user profiles is published, it gives all users access to the profile via the "Who is online" tool
+  and, through that, the option to request contact with that profile.
+  - This can be activated in your profile under "Profile and Privacy > Publish Profile > Visible for logged-in Users"
+    setting.
+  - This setting does not allow or prevent the contact request itself.
+- A contact can be used for two actions:
+  - You can add contacts to your mailing lists
+  - You can invite contacts to chatrooms
+- Furthermore, approved contacts are shown within the "Who is online?" tool
+- The component is not aware of the data being passed through mails or chats. It does not know anything about
+  the origin (i.e. course) of the content or whether it contains user related data or not.
+
+## Data being stored
+
+- Each contact request is stored, including the ID of the requesting user and the ID of the requested user, as well as a
+  timestamp of the request time. This does not require the consent of both target users.
+- When the requested user actively ignores the request, this data entry is added with an "ignore" flag.
+- After confirmation, each contact is stored, including the ID of the requesting user and the ID of the requested user as
+  well as a timestamp of the request time.
+- The storage of requests and approved contacts are separated.
+- Each contact added as an entry to a mailing list is stored by using the user_id.
+- Any user relations that originate from the contact is not related or depending on the active contact. There is no
+  traceable relation of contacts withing other components.
+
+## Data being presented
+
+- Contact data is presented within "Communication > Contacts" and within a separate section of the "Who is online?" tool.
+- The presentation includes a presentation of the user's profile (restricted depending on the user's profile publishing
+  settings) and control elements presenting the user's options for contact requests ("Confirm Contact"/"Ignore Contact").
+- Initially the options are restricted to requesting a contact.
+- The requesting user can cancel the request.
+- The requested user has the option to actively ignore or confirm the request.
+- Both contacts can unlink the contact without any further consent.
+- Each party can detect a change of the contact relation within its own contact options. The only exception to this
+  is the active ignoring of a request. The requesting user can not detect if his request was ignored.
```

Comment:
Lines 48-52: does this information belong there or more likely in General information? There is no information about presented data  in these lines.

Reply:
This line is not meant to target the functionality, but the active visible options in the dropdown which are a part of the presentation IMO since they imply certain states of requests.

I tried to clarify this with a more detailed writing. Please let me know what you think of it.

Reply:
Thank you for the detailed description, @iszmais ! 
Just for me to clarify: If an user is responding to a request (by choosing one of these options), then a corresponding data entry will be stored. That's why these options are important, did I understand it correctly? There isn't any personal data visible, but choosing the option triggers the personal data processing, doesn't it?

Reply:
@ksgribchenko exactly!
And further, the possible options imply the stored data.
E.g. If the option "Unlink" is visible, this implies that a contact relation to this user exists.
Therefore, it provides the same information as displaying the actual request state directly.